pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -30,21 +30,25 @@ def __lt__(self, requirement):
 
 def fix_requirements(f):
     requirements = []
-    before = []
```

Comment:
could even do `tuple(f)` (since tuples are smaller, faster, and best of all immutable!)